src/Data/NormalKey.purs

```diff
@@ -82,26 +80,37 @@ toApple key = key
 combinationToApple :: Set NormalKey -> Set NormalKey
 combinationToApple = toList >>> map toApple >>> fromList
 
-instance showKey :: Show Key where
-  show (Key s) = "(Key " ++ show s ++ ")"
+data NormalKeyView = Modifier Int | Other String | Enter
+
+toView :: NormalKey -> NormalKeyView
+toView (NormalKey "Control") = Modifier 0
+toView (NormalKey "Meta") = Modifier 1
+toView (NormalKey "Alt") = Modifier 2
+toView (NormalKey "Shift") = Modifier 3
+toView (NormalKey "Enter") = Enter
+toView (NormalKey s) = Other s
 
 instance eqNormalKey :: Eq NormalKey where
   eq (NormalKey x) (NormalKey y) = eq x y
 
+instance eqNormalKeyView :: Eq NormalKeyView where
+  eq (Modifier x) (Modifier y) = eq x y
+  eq (Other x) (Other y) = eq x y
+  eq Enter Enter = true
+  eq _ _ = false
```

Comment:
I think that `Ord` relies on `Eq`? https://github.com/purescript/purescript-prelude/blob/master/src/Prelude.purs#L639